docs/api/components/schemas.yaml

```diff
@@ -2512,3 +2512,99 @@ CollectionNames:
           - Starting
           - Stopping
           - Resyncing
+AllUserChannels:
+  description: |-
+    All user channels split by how they were assigned to the user and by keyspace.
+  type: object
+  properties:
+    admin_grants:
+      description: |-
+        Channels assigned to the user through the admin API by adding them to admin_channels.
+      type: object
+      additionalProperties:
+        x-additionalPropertiesName: channel
+        description: Channel name
+        type: object
+        properties:
+          Entries:
+            type: array
+            description: Start sequence to end sequence
+          UpdatedAt:
+            type: integer
+            description: Unix timestamp of last update
+          source:
+            type: string
```

Comment:
You can create an object for this, but it needs two additional levels of nesting for `scope` and `collection`. This applies to all objects here.